components/atom/tag/src/index.js

```diff
@@ -34,14 +34,15 @@ const filterKeys = (obj, listOfProps) =>
   }, {})
 
 const AtomTag = props => {
-  const {href, icon, onClick, size, responsive} = props
+  const {href, icon, onClick, size, responsive, className} = props
```

Comment:
Passing a `className` as prop is so dangerous because you're defining your component as "free" of styles, and our components has been defined regarding some "closed" styles. But it's true that this specific component should allow us to define some custom color variations, is that what you need @jordevo ?

Reply:
If so, you could do something like I've committed in other branch: https://github.com/SUI-Components/sui-components/commit/ed721ae0f7d4cd03f9cdb18b1f0d9be087134d49. This way any vertical could define in its own theme the types (variations) they need:

__THEME SASS file__
```css
$atom-tag-types: (
  'financed': (
    bgc: red,
    c: white
  ),
  'top': (
    bgc: blue,
    c: white
  )
);
```

__COMPONENT file__
```js
<AtomTag type='financed' />
```

How do you see this solution? @turolopezsanabria @juanmaguitar @carlosvillu @midudev @jcger

Reply:
plus 1 to this

Reply:
A very elegant solution. Yes, sir 👍

Reply:
I get your point, @kikoruiz ! thanks for your solution! 👏

Reply:
Alright, I'm going on with this solution, thanks!